mnefun/_mnefun.py

```diff
@@ -1458,7 +1598,8 @@ def _raw_LRFCP(raw_names, sfreq, l_freq, h_freq, n_jobs, n_jobs_resample,
         print('    Loading and filtering %d files.' % len(raw_names))
     raw = list()
     for rn in raw_names:
-        r = Raw(rn, preload=True)
+        with warnings.catch_warnings(record=True):
+            r = Raw(rn, preload=True, allow_maxshield=allow_maxshield)
```

Comment:
you can change this to `allow_maxshield='yes'` and then you don't need the `with` context manager (new feature in mne-python)